stress-example/nlp_search/app.py

```diff
@@ -0,0 +1,104 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+__version__ = '0.0.1'
+
+import click
+import os
+import sys
+
+from jina.flow import Flow
+from jina import Document
+
+NUM_DOCS = 50
+NUM_CHUNKS = 5
+QUERY_NUM_DOCS = 1
+TOP_K = 3
+REQUEST_SIZE = 4
+IMG_HEIGHT = 224
+IMG_WIDTH = 224
+
+
+def config(indexer_query_type):
+    parallel = 1
```

Comment:
But these are the same, right? So which one are you using when?

Reply:
True, I'm not using the others, I'll clean it up

Reply:
It seems they are both used. Line 26-27 you are setting the env. And then you are reading them in the yml